File: SMM.js
const twilio = require('twilio');

// Create authenticated API client as before
const client = twilio(
  process.env.TQ_TWILIO_ACCOUNT_SID,
  process.env.TQ_TWILIO_AUTH_TOKEN
);

// You'll be filling out some of the parameters yourself below.
// Take a look at the "from" parameter format and fill out the "to", 
// "mediaUrl" and "body" parameters. The "mediaUrl" parameter must be a 
// fully qualified URL to an image/some media you want to send
// with your message
client.messages
  .create({
    from: 'whatsapp:+14155238886',
    to: 'whatsapp:+12895521178',
    body: 'Matrix lecture',
    mediaUrl: 'https://www.math.tamu.edu/~dallen/m640_03c/lectures/chapter6.pdf',
  })
  .then(message => {
    console.log('Message is sent successfully!!');
  })
  .catch(error => {
    console.error('Looks like the Twilio API returned an error:');
    console.error(error);
  });

File: SMM.py
from twilio.rest import Client
import os
# Your Account Sid and Auth Token from twilio.com/console
account_sid = os.environ["account_sid"]
auth_token = os.environ["auth_token"]
client = Client(account_sid, auth_token)

message = client.messages.create(
    from_="whatsapp:+14155238886", # Your Sandbox number
    to="whatsapp:+12895521178", # Your own WhatsApp-enabled number
    body="Here's a picture of lion!",
    media_url="https://images.unsplash.com/photo-1573725342230-178c824a10f2?ixlib=rb-1.2.1&auto=format&fit=crop&w=651&q=80",
)

print("Message is sent successfully!!")
